utils/data.ts

```diff
@@ -50,7 +50,7 @@ export const transactionTabs: TransactionData = {
       lazlo: 'pending',
     },
     input: {
-      sats: '101,000',
+      sats: '100,000',
```

Comment:
I think this one stays as 101,000, right?

I sat down to review this PR then the baby woke up :sob:. Going to leave this single lonely comment here and try to continue my review tomorrow.